tests/ai_hub/utils.py

```
@@ -1068,3 +1068,14 @@ def get_latest_job_pod(admin_client: DynamicClient, job: Job) -> Pod:
     latest_pod = sorted_pods[0]
     LOGGER.info(f"Found {len(pods)} pod(s) for job {job.name}, using latest: {latest_pod.name}")
     return latest_pod
+
+
+def execute_authenticated_post(url: str, token: str, files: dict[str, tuple[str, str, str]]) -> dict[str, Any]:
+    """Execute an authenticated POST with multipart/form-data and return parsed JSON."""
+    headers = {"Authorization": f"Bearer {token}"}
+    LOGGER.info(f"Executing POST: {url}")
+    response = requests.post(url=url, headers=headers, files=files, verify=False, timeout=60)
```

Ruff (0.15.18)

[error] Probable use of `requests` call with `verify=False` disabling SSL certificate checks

(S501)